api/pkg/transformer/pipeline/testdata/upi/invalid_postprocess_output.yaml

```diff
@@ -0,0 +1,28 @@
+transformerConfig:
```

Comment:
For all invalid configurations, can you add inline comment which point to the part that make the config invalid?